examples/json-to-record/json_to_record.bal

```diff
@@ -0,0 +1,64 @@
+import ballerina/data.jsondata;
+import ballerina/io;
+
+type Book record {
+    string name;
+    string author;
+    int year;
+};
+
+public function main() returns error? {
+    json jsonContent = {
+        "name": "Clean Code",
+        "author": "Robert C. Martin",     
+        "year": 2008
+    };
+
+    // Convert the JSON value to a record type.
+    Book book1 = check jsondata:parseAsType(jsonContent);
+    io:println(book1);
+
+    string jsonStr = string `
```

Comment:
Imo, define this in module level will be more clear?. wdyt?

Reply:
Updated https://github.com/ballerina-platform/ballerina-distribution/pull/5294/commits/4431689f26e34b657e1c0b9fb8dde5f6179f4c37